tools/export-rtl/export-rtl.cpp

```diff
@@ -455,7 +455,18 @@ void WriteModData::writeSignalDeclarations(
           addExtraSignals(valueAndName.second, type.getExtraSignals());
         })
         .Case<IntegerType>([&](IntegerType intType) {
-          writeDeclaration(valueAndName.second, getRawType(intType), os);
+          // @jiahui17: this is a very bad hack to force the type to be an array
+          // when the array only has one element.
```

Comment:
Could you write something like:

- `getRawType` specifies `std_logic` instead of `std_logic_vector` when the bitwidth is 1.
- However, address signals should still be declared as `std_logic_vector`, even when their bitwidth is 1.